content/concepts/secure-comm/noise.md

```diff
@@ -6,5 +6,59 @@ aliases:
     - "/concepts/secure-comm/noise"
 ---
 
-<!-- ADD NOTICE -->
-Coming soon!
+## What is Noise?
+
+The [Noise Protocol Framework](https://noiseprotocol.org/) is a set of tools for creating
+secure communication protocols by combining cryptographic primitives into patterns with
+verifiable security properties. It provides a high level of flexibility, allowing users to
+choose from a range of cryptographic primitives and design patterns to suit their needs.
+However, it is up to the user to carefully consider the tradeoffs and make informed decisions
+to create a secure protocol.
```

Comment:
```suggestion
verifiable security properties.
```

libp2p already has made these choices. Users here don't have to think about this.

Reply:
Incorporated